x-pack/plugins/maps/public/components/global_time_checkbox.tsx

```diff
@@ -21,13 +21,21 @@ export function GlobalTimeCheckbox({ applyGlobalTime, label, setApplyGlobalTime
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalTime}
-        onChange={onApplyGlobalTimeChange}
-        data-test-subj="mapLayerPanelApplyGlobalTimeCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalTimeHelp', {
+          defaultMessage:
+            'When enabled, time filter from global search bar is applied to this layer"',
```

Comment:
GlobalTimeCheckbox is used 2 places, layer editor and join editor. As such, the phrase "this layer"should not be hard coded in the message.

How about "When enabled, results narrowed global time filter"